packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -312,6 +299,20 @@ export let BreadcrumbWithOverflow = ({
           noTrailingSlash={noTrailingSlash}
           {...other}
         >
+          {backItem?.href && (backItem?.label || backItem?.title) && (
```

Comment:
i don't think the `?.` operator is necessary since you're not chaining anything. you should get the same results with just `.`

Reply:
Probably true in 99% of cases. However, if only one breadcrumb is passed and it happens to be the current page then 271 will result in backItem being nullish.

Reply:
ahh ok i see. that makes sense. thanks for clarifying.